view_helpers.py

```diff
@@ -15,23 +16,27 @@
 from htk.utils import htk_setting
 from htk.utils import utcnow
 
+
 def render_to_response_custom(template_name, data=None, template_prefix=''):
     """Wrapper function for django.shortcuts.render_to_response
 
     Puts additional information needed onto the context dictionary
     """
-    if data is None:
-        data = {}
+    data = _data_processor(data, template_name, template_prefix)
+    response = render_to_response(template_name, data)
+    return response
 
-    # pre-render
-    data['javascripts'] = get_javascripts(template_name, template_prefix=template_prefix)
-    _build_meta_content(data)
-    _build_breadcrumbs(data)
 
-    # render
-    response = render_to_response(template_name, data)
+def render_custom(request, template_name, data=None, template_prefix=''):
+    """Wrapper function for django.shortcuts.render_to_response
```

Comment:
also, how is this function used / why is it needed?

Reply:
Okay going to replace `render_to_response` to `render`. For HTK it does the same thing as `render_to_response_custom` adding the same stuff to the context. But it is different in that it is django's `render` instead of `render_to_response`.

Reply:
@ajnassar 

it looks like `render` is the newer/better replacement for `render_to_response`

Per this [documentation](https://docs.djangoproject.com/en/1.11/topics/http/shortcuts/#render-to-response), its only difference is in also accepting the `request` as an argument, but otherwise all the same.

Can you put this function above? We should gradually start replacing everything with `render`